library/ajax/document_helpers.php

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
@bradymiller I thought you meant license. Why do I need to include a bunch of dependencies I don't need for this script. I get the need of a standard header but only where needed. Am I missing something?